python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -451,10 +462,20 @@ def check_valid_run_requests(self, run_requests: Sequence[RunRequest]):
     def _target(self) -> Optional[Union[DirectTarget, RepoRelativeTarget]]:
         return self._targets[0] if self._targets else None
 
+    @public  # type: ignore
+    @property
+    def job_name(self) -> Optional[str]:
+        return self.pipeline_name
+
     @property
     def pipeline_name(self) -> Optional[str]:
         return self._target.pipeline_name if self._target else None
 
+    @public  # type: ignore
+    @property
+    def op_selection(self) -> Optional[Sequence[str]]:
```

Comment:
Are we going to add an op_selection argument to `@sensor`? In the past I think we've considered just telling people to do the selection on the job instead. If that's the case, we shouldn't expose this.  Same for schedule.

Reply:
gotcha. sounds reasonable.